src/mon/OSDMonitor.h

```diff
@@ -249,6 +249,35 @@ class OSDMonitor : public PaxosService {
     FAST_READ_DEFAULT
   };
 
+  struct CleanUpmapJob : public ParallelPGMapper::Job {
+    CephContext *cct;
+    const OSDMap& osdmap;
```

Comment:
ParallelPGMapper::Job have a member 'Const OSDMap *osdmap', so we can use that one?